api/client/commands.go

```diff
@@ -737,16 +737,19 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 
 	if *attach || *openStdin {
 		if cmd.NArg() > 1 {
+			close(hijacked)
```

Comment:
I have tested it before, found that can not simply add defer close(hijacked) like this, it will affect the normal process.